news/2543-lines-columns.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+* Clean out ``$LINES`` and ``$COLUMNS`` if set, preventing some programs from drawing weird.
```

Comment:
*weirdly :)

Reply:
You spend too much time editing papers. 😛

Reply:
Hahah tell me about it :)